content/blog/2020-03-12-native-classroom/index.Rmarkdown

```diff
@@ -0,0 +1,93 @@
+
+For this workshop, I've chosen the route of letting students choose among datasets on public health and social justice issues. These topics are too widely pertinent to Native peoples, e.g. diabetes ^[[Call to action: A new path for improving diabetes care for Indigenous peoples, a global review](https://www.sciencedirect.com/science/article/pii/S0168822716302467)], cancer ^[[Cancer in Indigenous Populations](https://www.thelancet.com/series/cancer-in-indigenous-populations)], suicide ^[[The Native American Identity Crisis and the Rise of Suicide](https://blog.nativehope.org/february-spotlight-native-american-identity)], and gun crime ^[[Gun Violence Has a Major Impact on Native Communities in the United States](https://www.teenvogue.com/story/gun-violence-has-a-major-impact-on-native-communities-in-the-united-states)]. Now, no group is homogenous and it is inaccurate to think that every member of a marginalized group is interested in social justice; however, these topics do draw the interest of my students. They are also issues that the power of data literacy can help students solve in their lifetimes. In the future, I plan to incorporate datasets on other contemporary issues, such as social media usage (similar to [Girls Inc of NYC](https://www.elsevier.com/connect/teaching-data-science-through-the-lens-of-social-justice-issues)), as the current hard-hitting topics may turn-off some students. The provided datasets serve as the basis for group projects, in which students hone and analyze their research questions.
+
+To create a safe and engaging classroom structure, I take a few different abstract and concrete steps...
+
+In the abstract, I first understand that for many Native Americans, and other historically marginalized populations, community and storytelling are a central part of life, playing massive roles in individual identity. The culture of many classrooms does not always value communities and their stories in the same way... Western higher education was established in a highly individualistic culture, whose Protestant Christian roots emphasize dutiful and successful work (not necessarily community) as a core component of individual identity ^[[The Protestant Ethic and Higher Education in America](https://www.jstor.org/stable/pdf/1050005.pdf?refreqid=excelsior%3A2bea6bf49df970ca4232f691609d8547)]. This culture has been introduced to the world too often via painful and dehumanizing experiences, like colonization, conquest, genocide, and slavery ^[[Higher Education in the United States](https://education.stateuniversity.com/pages/2044/Higher-Education-in-United-States.html)]. So, the elevation of the individual through hustle and perfection, which is a common ambition of higher education, does not always feel familiar to students, and if familiar, it does not always feel safe. For many students, instead, a focus on community and connections to community makes classroom material much more accessible. 
```

Comment:
1. I don't understand what the ellipsis is for.
2. I would remove "Protestant Christian" - the Catholic and Jewish influences on Western European and American schooling are also important.

Reply:
I would remove the sentence "This culture has been introduced..." and add the citation [7] to the previous citation [6].